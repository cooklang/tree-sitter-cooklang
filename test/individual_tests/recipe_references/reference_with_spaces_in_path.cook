Top with @./Sauces/Super Sweet{1%portion}.