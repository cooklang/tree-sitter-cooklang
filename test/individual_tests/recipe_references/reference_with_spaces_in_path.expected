(recipe 
  (step 
    (ingredient 
      (recipe_reference) 
      (amount (quantity) (units)))))